corehq/apps/userreports/reports/util.py

```diff
@@ -26,3 +36,72 @@ def has_location_filter(view_fn, *args, **kwargs):
         getattr(filter_, 'location_filter', False)
         for filter_ in report.ui_filters
     )
+
+
+class ReportExport(object):
```

Comment:
I wasn't sure where best to put this, so I put it in `util`. It looks like the convention in this module (`userreports.reports`) is to create a new file per object. Should I put this in a `report_object.py` file instead?